`empty-args` rule labeled `no-empty-args`?

When running (with `-q -v`) against an existing project's `.scss` files, among other warnings I see many labeled "no-empty-args":

```
...a.scss
  11:19  warning  Parenthesis should be removed  no-empty-args

...b.scss
  122:19  warning  Parenthesis should be removed  no-empty-args

...c.scss
  9:18  warning  Parenthesis should be removed  no-empty-args

...d.scss
   4:25  warning  Parenthesis should be removed  no-empty-args
  18:23  warning  Parenthesis should be removed  no-empty-args
  24:10  warning  Parenthesis should be removed  no-empty-args
  29:10  warning  Parenthesis should be removed  no-empty-args
  41:10  warning  Parenthesis should be removed  no-empty-args
  48:10  warning  Parenthesis should be removed  no-empty-args
  56:10  warning  Parenthesis should be removed  no-empty-args
  64:10  warning  Parenthesis should be removed  no-empty-args
  76:18  warning  Parenthesis should be removed  no-empty-args
```

(Those lines are all definitions or uses of mixins with empty parens after the mixin name.)

Our config file is using the "merge with default rules" option. I try to mark those as not being warning-worthy by adding `no-empty-args: 0` to my config file, but I still see the warnings. However, if I add `empty-args: 0` the warnings disappear.

Should the warning messages have "empty-args" as the rule name at the end of the line?

--- a/lib/rules/empty-args.js
+++ b/lib/rules/empty-args.js
@@ -3,7 +3,7 @@
 var helpers = require('../helpers');
 
 module.exports = {
-  'name': 'no-empty-args',
+  'name': 'empty-args',
   'defaults': {
     'include': false
   },

